src/Control/Carrier/FossaApiClient.hs

```diff
@@ -11,11 +11,30 @@ import Control.Carrier.Simple (SimpleC, interpret)
 import Control.Effect.Debug (Debug)
 import Control.Effect.Diagnostics (Diagnostics)
 import Control.Effect.FossaApiClient (FossaApiClientF (..))
-import Control.Effect.Lift (Lift)
+import Control.Effect.Lift (Lift, sendIO)
+import Data.Default.Class (def)
 import Fossa.API.Types (ApiOpts)
+import Network.Connection (TLSSettings (TLSSettingsSimple, settingClientSupported))
+import Network.HTTP.Client (Manager, ManagerSettings, newManager)
+import Network.HTTP.Client.TLS (mkManagerSettings)
+import Network.TLS (EMSMode (AllowEMS), Supported (supportedExtendedMainSecret))
+
+-- TODO: Remove the ReaderC Manager layer.
+-- This was created so that we can use AllowEMS for a few older servers that still require it.
+-- In 10/24 we've said we'll be reverting this change.
+-- After fixing the related errors, you should also be able to remove these deps from spectrometer.cabal:
+--  1. crypton-connection
+--  2. http-client-tls
+--  3. data-default-class
```

Comment:
Please be super clear with dates here, I was a bit confused by this:
```suggestion
-- TODO: Remove the ReaderC Manager layer.
-- This was created so that we can use AllowEMS for a few older servers that still require it.
-- As of 2024-10-24 we are free to revert this change, but it must stay until then.
-- After fixing the related errors, you should also be able to remove these deps from spectrometer.cabal:
--  1. crypton-connection
--  2. http-client-tls
--  3. data-default-class
```